avocado/job.py

```diff
@@ -221,9 +221,6 @@ def _run(self, urls=None, multiplex_files=None):
                 if url.startswith(os.path.pardir):
                     url = os.path.abspath(url)
                 params_list.append({'id': url})
-        else:
```

Comment:
From the API perspective, This else condition still holds. Like, if you instantiate `j = Job()` and then `j.run()`, the current behavior will raise `OptionValidationError: Empty test ID. A test path or alias must be provided`.
When you remove it, after run() you'll get `TypeError: 'NoneType' object is not iterable`, which is wrong. Right? So let's keep it.